enclave_manager/avalon_enclave_manager/singleton/enclave_manager.py

```diff
@@ -237,7 +218,13 @@ def _process_work_order_by_id(self, wo_id):
     def _execute_work_order(self, input_json_str, indent=4):
         """
         Submits workorder request to Worker enclave and retrieves the response
+
+        Returns :
+            json_response - A JSON formatted str of the response receieved from
+                            enclave. Errors are wrapped in JSON if exceeptions
+                            have occured.
```

Comment:
s/occured/occurred/
s/receieved/received/
s/exceeptions/exceptions/
s/enclave./the enclave./